package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "async": "^0.9.0",
     "bcryptjs": "^2.1.0",
+    "bluebird": "^2.9.9",
```

Comment:
Didn't notice the older version of bluebird here. As suggested by @bajtos, please use 3.1.1 (latest on NPM - I checked for you).